requirements/prd.txt

```diff
@@ -1,7 +1,5 @@
 fire==0.7.0
 jax==0.5.3
-numpy==2.2.4
+numpy<2.2.0
```

Comment:
😬 What's the story?

Reply:
tensorflow was complaining about the `numpy` version.

I am going to double check that I didn't get stuck in some sort of accidental depency hell. before I mark ready for review.

Reply:
We may want to keep the `<=`, different versions of python use different versions of `numpy`. See #152

Reply:
The ``requirements`` are frozen by definition. They need to be exactly what we want and use, including that for the given version of python (that we only loosely specify in the docs and ``pyproject.toml``). This is one reason I prefer using conda env yaml files over pip, as you can specify the python version, however, with pip it's a chicken and egg situation.

These do need to be frozen for py3.11. Though feel free to upgrade our python version as specified in the ``README.md`` and ``pyproject.toml``.

The deps in ``pyproject.toml`` are loose by definition and there we can spec min/max versions.

Reply:
Ah makes sense. Thanks for explaining, I'll update.

Reply:
And I totally take it back, you can spec the python version in the pip requirements file. So we should def do that in another PR etc.

Reply:
haha sounds good.